src/Geno/C++/GUI/Widgets/TextEdit.h

```diff
@@ -18,45 +18,194 @@
 #pragma once
 #include <Common/Macros.h>
 #include <Common/Texture2D.h>
-
 #include <filesystem>
+#include <imgui.h>
 #include <string>
 #include <vector>
 
-class  Drop;
+class Drop;
 struct ImGuiTabBar;
 
 class TextEdit
 {
-public:
+   public:
```

Comment:
It's not supposed to look like this. Did you use the latest version of the .clang-format file?

Reply:
I think vs did some magic on it's own, it went back if I run clang-format